tests/test_parser.py

```diff
@@ -5,6 +5,22 @@
 from myia.utils.info import enable_debug
 
 
+def test_posonly_args():
+    def f1(a, b, c, d, e):  # pragma: no cover
+        pass
+
+    g1 = parse(f1)
+    assert len(g1.parameters) == 5
+    assert g1.posonly == 0
```

Comment:
There are already tests for normal parameters so I don't think we need to test it again.

Also, it would be nice if this test would go with the other `test_def` below.